vehicle-python/pyproject.toml

```diff
@@ -15,8 +15,15 @@ readme = 'README.md'
 license = 'MIT'
 license-files = [ "LICENSE" ]
 dynamic = ["version"]
-requires-python = ">=3.9,<3.14"
-dependencies = ["typing_extensions >=4.6,<5", "jaxtyping>=0.2, <0.3"]
+requires-python = ">=3.10,<3.14"
+dependencies = [
+    "typing-extensions>=4.6,<5",
+    "jaxtyping>=0.2, <0.3",
+    "tensorflow>=2.20.0",
```

Comment:
Shouldn't `tensorflow` and `torch` be part of `[project.optional-dependencies]` dependencies rather than `[project]`. We don't want to force users to install both right?